sycl/test/extensions/annotated_usm/fake_properties.hpp

```diff
@@ -0,0 +1,446 @@
+//==--- mock_compile_time_properties.hpp -  Mock compile-time properties ---==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// Since we do not currently have any compile-time properties this header cheats
+// a little by defining its own compile-time properties. These are intended for
+// testing and users should not replicate this behavior.
+
```

Comment:
I must be misunderstanding or overlooking something.  Why is this test here? How does it tie into USM annotations?

Reply:
nevermind - I see